server.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+require('dotenv').config();
+const app = new express();
+const port = process.env.PORT || 5000;
+
+app.use(express.static('public'));
+
+app.get('/', (req, res) => {
+  res.sendFile(__dirname + '/');
+});
+
+app.get('/config.js', (req, res) => {
```

Comment:
What do you think about this vs. using a templating language (I've used Pug before) and passing as a variable?

Main reason I'm asking this is because in the next 1-2 days we're going to need to be dynamically rendering meta tags in the <head> depending on the user's locale

Another benefit is eliminating a request for config.js

Downside is probably that it's another thing for people to learn

Reply:
Unfortunately outside my realm of knowledge but happy to do it if you can help me with this